.github/workflows/suite-native_production_ci.yml

```diff
@@ -20,6 +20,8 @@ jobs:
     name: Build and release iOS
     environment: production
     runs-on: ubuntu-latest
+    env:
+      EXPO_PUBLIC_ENVIRONMENT: production
```

Comment:
You added it only for iOS, please add it to Android flow too

Reply:
https://github.com/trezor/trezor-suite/pull/11720/commits/3dfc01061d1bf8a920aaacd9a2605ef516b56916